### PR TITLE
MUI. Material `5.14.6` & Icons `5.14.6` & Lab `5.0.0-alpha.141` [generated]

### PR DESCRIPTION
**Pull request by aerialist7**

### CODE REVIEW

File: kotlin-mui/src/jsMain/generated/mui/base/Modal.types.kt [hunk 1]
```diff
@@ -13,7 +13,7 @@ external interface ModalOwnProps : react.Props {
     /**
      * A single child content element.
      */
-    var children: dynamic /* react.ReactElement<*> */
+    var children: react.ReactNode?
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/Modal.types.kt:16]: Missed override

File: kotlin-mui/src/jsMain/generated/mui/material/Modal.kt [hunk 2]
```diff
@@ -41,6 +42,11 @@ external interface ModalProps :
      */
     var BackdropProps: BackdropProps?
 
+    /**
+     * A single child content element.
+     */
+    var children: react.ReactNode?
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/material/Modal.kt:48]: Missed override

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-mui/src/jsMain/generated/mui/base/Modal.types.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/Modal.types.kt:16]: No :(

File: kotlin-mui/src/jsMain/generated/mui/material/Modal.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/material/Modal.kt:48]: Yeas, "VIRTUAL_MEMBER_HIDDEN" lover detected :)

**Pull request merged**